### DISCUSSION TITLE
What OS(s) is version Podman v3.2+ available on?

### DISCUSSION THREAD

**Discussion by robbie-hunt** [containers/podman#10891] [General]
Where is v3.2/the latest version of Podman available to use? I would like to use the docker-compose feature of Podman, although I can't seem to run more than v3.0.1. I'm running DietPi 7.3.2 x64 (upgraded to Bullseye) on a Raspberry Pi 4B.

**Comment by rhatdan** [comment]
@lsm5 PTAL
